presto-salesforce/pom.xml

```diff
@@ -0,0 +1,216 @@
+<?xml version="1.0"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.prestosql</groupId>
+        <artifactId>presto-root</artifactId>
+        <version>339-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>presto-salesforce</artifactId>
+    <description>Presto - Salesforce Connector</description>
+    <packaging>presto-plugin</packaging>
+
+    <properties>
+        <air.main.basedir>${project.parent.basedir}</air.main.basedir>
+        <air.check.skip-duplicate-finder>true</air.check.skip-duplicate-finder>
+
+        <google-oauth-client.version>1.25.0</google-oauth-client.version>
+        <commons-lang3.version>3.8</commons-lang3.version>
+        <commons-collections4.version>4.1</commons-collections4.version>
+
+        <junit.version>4.12</junit.version>
+        <hamcrest.version>1.3</hamcrest.version>
+
+        <maven-compiler-plugin.version>3.7.0</maven-compiler-plugin.version>
+        <versions-maven-plugin.version>2.5</versions-maven-plugin.version>
+        <maven-surefire-plugin.version>2.21.0</maven-surefire-plugin.version>
+        <maven-source-plugin.version>3.0.1</maven-source-plugin.version>
+        <maven-jar-plugin.version>3.1.0</maven-jar-plugin.version>
+        <maven-assembly-plugin.version>3.1.0</maven-assembly-plugin.version>
+        <maven-deploy-plugin.version>2.8.2</maven-deploy-plugin.version>
+        <salesforce-soql-parser.version>2.0</salesforce-soql-parser.version>
+        <force-partner-api.version>43.0.0</force-partner-api.version>
+        <mapdb.version>3.0.5</mapdb.version>
+        <opencsv.version>4.2</opencsv.version>
+        <xstream.version>1.2.2</xstream.version>
+
+    </properties>
+
+    <repositories>
+        <repository>
+            <id>mulesoft-releases</id>
+            <name>MuleSoft Releases Repository</name>
+            <url>http://repository.mulesoft.org/releases/</url>
+            <layout>default</layout>
+        </repository>
```

Comment:
Are the artifacts in maven central? It's not a good idea to depend on third-party repositories. It makes the build more brittle and makes it hard for people that are using a maven proxy in their organizations.

Reply:
Good point. Let me check on that.